server/utils/sequelize.js

```diff
@@ -0,0 +1,17 @@
+import  Sequelize  from 'sequelize';
+
+
+
+
+
+const sequelize = new Sequelize('postgres', 'postgres', 'mypass', { 
```

Comment:
My local postgres has different username and passwords than yours. Can we move those strings into environment file? So everyone can set up their own username and password